map/src/main/kotlin/edu/artic/map/SearchObjectDetailsFragment.kt

```diff
@@ -0,0 +1,164 @@
+package edu.artic.map
+
+import android.os.Bundle
+import android.view.View
+import com.bumptech.glide.Glide
+import com.fuzz.rx.*
+import com.jakewharton.rxbinding2.view.clicks
+import com.jakewharton.rxbinding2.widget.text
+import edu.artic.analytics.ScreenCategoryName
+import edu.artic.db.models.ArticObject
+import edu.artic.media.audio.AudioPlayerService
+import edu.artic.media.ui.getAudioServiceObservable
+import edu.artic.viewmodel.BaseViewModelFragment
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.rxkotlin.withLatestFrom
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.fragment_search_object_details.*
+import kotlin.reflect.KClass
+
+/**
+ * Displays the object details of the searched item.
```

Comment:
We really ought to have some disambiguation with `MapObjectDetailsFragment`.

Reply:
Are you talking about the name of the fragment or the functionality? Once I integrate the map annotation it will have more features (e.g. cafe carousel etc.) than the `MapObjectDetailsFragment`.